src/wallet/db.cpp

```diff
@@ -53,7 +53,7 @@ std::map<std::string, BerkeleyEnvironment> g_dbenvs GUARDED_BY(cs_db); //!< Map
 
 bool WalletDatabaseFileId::operator==(const WalletDatabaseFileId& rhs) const
 {
-    return memcmp(value, &rhs.value, sizeof(value)) == 0;
+    return memcmp(value, rhs.value, sizeof(value)) == 0;
```

Comment:
Might I ask, why is this a `memcmp` at all, when `value` is basically a `u_int8_t`?

Wouldn't return `rhs.value == value` be simpler?

Reply:
It's an array

https://github.com/bitcoin/bitcoin/blob/master/src/wallet/db.h#L30

Reply:
😬sorry about that.

Reply:
No problem :smile: